public/data/anushatomar13.json

```
@@ -0,0 +1,13 @@
+                 {
+ "name": "Anusha Tomar",
+ "location": "New Delhi, India",
+ "bio": "I am a Google WE Scholar, Harvard WECode Scholar, HPAIR Delegate, MLSA, and an open-source enthusiast.",
+ "avatar": "https://github.com/anushatomar13.png",
+ "portfolio": "https://github.com/anushatomar13",
+ "skills": ["Machine Learning", "Web Development", "Open-source enthusiast"],
+ "social": {
+   "GitHub": "https://github.com/anushatomar13",
+   "Twitter": "",
+   "LinkedIn": "https://www.linkedin.com/in/anushatomar"
+ }
+},
```

```suggestion
}
```
Remove last comma from your profile file.